phpBB/phpbb/search/fulltext_sphinx.php

```diff
@@ -136,7 +136,8 @@ public function __construct(&$error, $phpbb_root_path, $phpEx, $auth, $config, $
 		$this->auth = $auth;
 
 		// Initialize \phpbb\db\tools\tools object
-		$this->db_tools = new \phpbb\db\tools\tools($this->db);
+		global $phpbb_container; // TODO inject into object
```

Comment:
We can't do it in the moment, because all search classes need the same constructor...
so this is another task